prov/usnic/src/fi_ext_usnic.h

```diff
@@ -42,7 +42,7 @@
 #include <stdint.h>
 #include <net/if.h>
 
-#define FI_PROTO_RUDP 100
+#define FI_PROTO_RUDP (100 | FI_PROV_SPECIFIC)
```

Comment:
> So you're suggesting that `FI_PROV_SPECIFIC` be changed to `(1u << 31)` and `FI_PROTO_RUDP` be changed to `(100u | FI_PROV_SPECIFIC)`?

Yes.

> We could always make them `(UINT32_C(1) << 31)` as well.

We *could*, but I think the `u` suffix on the constant is more readable/obvious to others.